packages/frontend/src/build/buildPages.ts

```diff
@@ -15,7 +15,7 @@ main().catch((e) => {
 async function main() {
   const env = process.env.DEPLOYMENT_ENV ?? 'production'
   const config = getConfig(env)
-  const apiMain = await fetchApiMain(
+  const apiMain = await fetchTvlApi(
```

Comment:
`apiMain` -> `tvlApiResponse`. This rename should propagate in the entire codebase